src/components/nav/components/desktop/components/action_bar/__snapshots__/index.test.tsx.snap

```diff
@@ -31,14 +31,14 @@ exports[`screen: Nav/ActionBar displays network 1`] = `
           </svg>
         </div>
         <input
-          aria-label="common:searchBarPlaceholderDtag"
+          aria-label="searchBarPlaceholderDtag"
```

Comment:
Hi @MonikaCat , did all these kinds of i18n locale changes in snap files happen after you ran `npm run test -- -u`?

Reply:
Hi @yayay927 yes that's right, all of the present snap files have been updated after I run yarn test cmd

Reply:
@MonikaCat Oh I see